example_site/src/components/Collections.js

```diff
@@ -31,45 +32,124 @@ function Template({ template }) {
   );
 }
 
+function DurationFilter({
+  filter,
+  templateFilter,
+  setTemplateFilter,
+}) {
+  const [buttonStatus, setButtonStatus] = useState(true);
```

Comment:
What exactly is `buttonStatus` controlling?

Reply:
Whether the filter is applied or not - I agree this is a vague variable name. Renamed to filterApplied, setFilterApplied.